store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -77,6 +85,24 @@ public void testWriteAndReadFiles() throws IOException, InterruptedException {
     Assert.assertEquals(i, 100);
 
     reader.close();
+
+    // Read again
+    CarbonReader reader2 = CarbonReader
+        .builder(path, "_temp")
+        .projection(new String[]{"name", "age"})
```

Comment:
Add a test case of two sequential reads but without closing the 1st reader, 2nd reader starts.